climakitae/core/data_interface.py

```diff
@@ -1814,6 +1825,7 @@ def get_data(
     cached_area=["entire domain"],
     area_average=None,
     time_slice=None,
+    station=None,
```

Comment:
Is it possible to rename this to `stations`? Since there are potentially multiple stations passed in?

Reply:
`station` does make sense for a singular station, but I think `stations` can make it more clear to the user that you _can_ pass multiple stations in, while `station` does not

Reply:
fixed that, thanks!